rpcserver.go

```diff
@@ -3948,8 +3944,7 @@ func (s *rpcServer) jsonRPCRead(w http.ResponseWriter, r *http.Request, isAdmin
 	if err != nil {
 		errMsg := fmt.Sprintf("error reading JSON message: %v", err)
 		errCode := http.StatusBadRequest
-		http.Error(w, strconv.FormatInt(int64(errCode), 10)+" "+errMsg,
-			errCode)
+		http.Error(w, strconv.Itoa(errCode)+" "+errMsg, errCode)
```

Comment:
I'd have to benchmark to verify, but I seem to recall reading that `fmt.Sprintf` is _much_ slower than the above.  I don't have a strong opinion on changing it however.  If you (and others) would prefer `fmt.Sprintf` instead there, I can change it.